src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -99,30 +107,35 @@ export const RegistrationFormActions = ({
       <Box
         sx={{
           display: 'grid',
-          gridTemplateAreas: "'back-button support-button save-button'",
-          gridTemplateColumns: '1fr 1fr 1fr',
+          gridTemplateAreas: {
+            xs: "'back-button . action-buttons' 'support-button support-button support-button'",
+            md: "'back-button  support-button  action-buttons'",
+          },
+          gridTemplateColumns: { xs: 'auto auto auto', md: '1fr 1fr 1fr' },
```

Comment:
Lurer på om jeg ville tatt denne ned til `sm`, da den bruker rader selv omden har ganske god plass nå?
```suggestion
            sm: "'back-button  support-button  action-buttons'",
          },
          gridTemplateColumns: { xs: 'auto auto auto', sm: '1fr 1fr 1fr' },
```

Reply:
Support-knappen blir litt chunky på et punk da, så tenkte det bare bedre at den holdt formen sin. Men ingen sterke meninger på dette.
![Screenshot 2024-08-22 at 10 12 50](https://github.com/user-attachments/assets/e98632ef-2867-4f25-bc04-700e83c44395)

Reply:
Skjønner, your call!